.github/workflows/build.yaml

```diff
@@ -0,0 +1,23 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
```

Comment:
This looks dangerous.  We don't want to build the containers whenever *this* repo is updated; we want to build whenever it's triggered by a new weekly being available.  I would change this to `workflow_dispatch`.

Note that Jenkins would not use this right now; as far as I can tell, it directly builds from the Dockerfile.